### ISSUE TITLE
No LND peers after power outage

### ISSUE THREAD

**Issue by yureqq** [raspiblitz/raspiblitz#2436]
Three days ago I experienced a power outage and my node stopped booting properly.

I downloaded a fresh new image of Raspiblitz and re-flashed the SD card. Upon on boot a screen prompted saying that a wallet and channels were found in HDD. It restored from there. 

Bitcoin succesfully connected to some peers and synced the blockchain but LND showed 0/11 channels and 0 peers. I then waited several hours without success. 

Right now, at MENU > INFO, LND shows 'Waiting for Chain Sync' even though it's synced already. If I restart LND then it'll show 0/11 channels and 0 peers at INFO screen.

Debug logs: https://termbin.com/3m4o
LND logs: https://termbin.com/uu0w
Bitcoin logs: https://termbin.com/f11x

**Comment by yureqq** [comment]
Last night I tried the `RESET-ALL` option. Copied blockchain from the desktop via LAN and restored on-chain funds and channels following [this](https://github.com/rootzoll/raspiblitz#lndrescue-lnd-targz-backupfile-best) method. And I got stuck on the same problem: no LND peers. But now the loop is reproducible and predictable.  

That's the line before it stops, cools a bit, and starts again.
  `2021-07-24 15:53:44.427 [ERR] LTND: unable to start server: unable to fetch node: 03179ae29181f2626812b48e11cf30e35cac7b707947bca0bf5e7b893016a5c3d5, unable to find node`

  `2021-07-24 15:53:44.427 [INF] RPCS: Stopping RPC Server`


LND logs: https://termbin.com/mr5a
